### DISCUSSION TITLE
What is the purpose of Unauthenticated Users

### DISCUSSION THREAD

**Discussion by vnetonline** [oqtane/oqtane.framework#4767] [General]
I am creating a free course, Let's Explore Oqtane Framework, and I need to explain what the purpose of unauthenticated users is can you please explain @sbwalker 

The course will be available on [LearnOqtane.com](https://LearnOqtane.com)

![image](https://github.com/user-attachments/assets/76bcad98-5422-4d31-9479-aa25a991f1fb)

Not a Member? [Join Now](https://LearnOqtane.com/join-now) for Free

**Comment by sbwalker** [comment]
Are you referring to the Unauthenticated Users pseudo-role which is displayed in the Permission grid when setting Page or Module settings?

**Comment by vnetonline** [comment]
@sbwalker Yes I am referring to the Role which appears in the Role
Management and then is visible in the Page and Module settings in the permission grid.

**Comment by sbwalker** [comment]
Sometimes there is content in a site which is only applicable to a user who is not authenticated. Imagine a module with content that says "You must login to view the content on this page". You would not want this module to be displayed for users who are already logged in. So in this case you would set the module's View permissions to allow Unauthenticated Users only (not All Users or Registered Users).

**Reply by Raceeend** [reply]
How beautiful and simple. Stupid of me not to think of this. Show the login module on the welcome page if you are not logged in, but not if you are already logged in (message: you are already logged in).
